### DISCUSSION TITLE
Does SwarmUI supports LoHa or Lycoris in general?

### DISCUSSION THREAD

**Discussion by 8bit-boom** [mcmonkeyprojects/SwarmUI#480] [Q&A]
Hello, I trying to use my LoHa's but I get error "ComfyUI execution error: self must be a matrix".

**Comment by mcmonkey4eva** [comment]
yes, most common LoRA variants should work fine. If something's not working, post an issue or join the Discord and post in the help-forum channel, either way including the output of the server->logs->pastebin button and a link to where you got the relevant lora from
